tests/unit/panther_analysis_tool/Pipfile

```diff
@@ -0,0 +1,11 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+
+[dev-packages]
+
+[requires]
+python_version = "3.9"
```

Comment:
Why do the unit tests need a pip file?

Reply:
Sorry, the random file that "git add ." added to my commit. Deleted